pcdsdevices/lodcm.py

```diff
@@ -62,31 +68,27 @@ class LODCM(InOutRecordPositioner):
     the mono line, onto both, or onto neither.
 
     This positioner only considers the h1n and diagnostic motors.
-%s
+
+    Parameters
+    ----------
+    prefix: ``str``
+        The PV prefix
+
+    name: ``str``, required keyword
+        The name of this device
+
     main_line: ``str``, optional
         Name of the main, no-bounce beamline.
 
     mono_line: ``str``, optional
         Name of the mono, double-bounce beamline.
     """
-    __doc__ = __doc__ % basic_positioner_init
-
-    state = Cpt(EpicsSignal, ':H1N', write_pv=':H1N:GO', kind='hinted')
-    readback = FCpt(EpicsSignalRO, '{self.prefix}:H1N:{self._readback}',
-                    kind='normal')
-
+    h1n = Cpt(H1N, ':H1N', kind='hinted')
```

Comment:
I always forget the rules... a hint of my hinted component is my hint?

Reply:
I think all the subhints end up bubbling up somehow... I can't remember either.